crypto/curve25519/curve25519.c

```diff
@@ -4520,13 +4521,9 @@ void GFp_x25519_sc_muladd(uint8_t *s, const uint8_t *a, const uint8_t *b,
   s[31] = (uint8_t)(s11 >> 17);
 }
 
-
-/* Prototype to avoid -Wmissing-prototypes warnings. */
-void GFp_ed25519_scalar_mask(uint8_t a[32]);
-
-void GFp_ed25519_scalar_mask(uint8_t a[32]) {
+void GFp_x25519_scalar_mask(uint8_t a[32]) {
```

Comment:
> Agreed, I've made a note to do this.

That is: I changed the `GFp_x25519_scalar_mask()` to `GFp_curve25519_scalar_mask()`, and will change the others separately.